packages/ra-core/src/reducer/admin/resource/list/cachedRequests.ts

```diff
@@ -35,6 +39,17 @@ const cachedRequestsReducer: Reducer<State> = (
         // force refresh
         return initialState;
     }
+    if (action.type === SOFT_REFRESH_VIEW) {
```

Comment:
This is the key difference: a soft_refresh doesn't empty the cached requests, only invalidates them